frontend/packages/dev-console/src/components/topology/shapes/BuildSuccessIcon.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { OutlinedCheckCircleIcon } from '@patternfly/react-icons';
+import { global_success_color_100 as successColor } from '@patternfly/react-tokens';
+
+export type BuildSuccessIconProps = {
+  className?: string;
```

Comment:
`className` is unnecessary. We're the only consumers of this component and you are not using `className`.
Safe to remove it.

Reply:
removed the BuildSuccessIcon altogether and use OulinedCheckCircleIcon directly in the decorator.